### DISCUSSION TITLE
Why do some photos have an estimated place?

### DISCUSSION THREAD

**Discussion by jean-louis67** [photoprism/photoprism#2212] [General]
Hello

I just uploaded 7 photos sent by a friend.
For most of them a (false) place is indicated. I don't understand where these different places come from.

Indeed, there are no GPS coordinates in the image.

When loading, we also have for each of these photos a message of this nature:

`time="2022-03-31T06:02:51Z" level=info msg="metadata: failed parsing GPS coordinates in IMG_4807.JPG (exif)"`

Looking at the database, we find this:
![image](https://user-images.githubusercontent.com/65403912/161005032-e41f61d1-29ec-4bed-892b-474ac924aa12.png)

By discovering this place_src column, I find that for 504 photos among 10400 we have place_src='estimate'
For most of my photos, place_src is 'meta' and for a few 'manual', values I understand.

What is the criterion that makes the estimate value chosen and how is the estimate then made?

And also, how to delete the estimate that does not fit? Does this "estimate" character appear anywhere in the application?

Thank you in advance for your opinion,

Jean-Louis

**Comment by graciousgrey** [comment]
Hello :)

For photos without GPS information PhotoPrism estimates a location based on photos taken around the same time that have GPS coordinates. You can turn this off in settings: https://docs.photoprism.app/user-guide/settings/library/#estimates

**Reply by lastzero** [reply]
Source Code: 
- https://github.com/photoprism/photoprism/blob/develop/pkg/geo/movement.go#L196

**Reply by lacrimal** [reply]
I've fallen victim to this behavior 🙂 My wife and I sync our photos from our phones into separate folders, which are then indexed by PhotoPrism. The result: her photos taken at home are being assigned my GPS data from a trip — about 100 km away.

It seems like the mechanism that infers or propagates location data might be a bit too aggressive or broad in scope.
It also looks that on most common version re indexing not brings any changes to already estimated files.

**Comment by jean-louis67** [comment]
Thank you Theresa and Michael so much for your responses.

I no longer remembered this parameter (which I had probably translated in french 2 years ago...)

So I disabled this feature

I then made many unsuccessful attempts to recover my photos without the location estimate
The one that worked:

- Delete photos then re-index
- Deleting photos from the photos table
- Deleting sidecar files
- Restarting PhotoPrism
- Adding photos and indexing
Good exercise!
A search function that would search for photos with estimated location as well as a delete estimated location function would come in handy as I intend to review my 500 affected photos.

Anyway, thanks again for your quick response!
